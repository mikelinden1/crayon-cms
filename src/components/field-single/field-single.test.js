import FieldSingle from './field-single';

it('FieldSingle matches snapshot', () => {
    const props = {
        name: 'test-field',
        actions: {
            setItemProp: () => {},
            validateOnChange: () => {}
        }
    };

    const wrapper = shallow(<FieldSingle {...props} />);
    expect(wrapper).toMatchSnapshot();
});