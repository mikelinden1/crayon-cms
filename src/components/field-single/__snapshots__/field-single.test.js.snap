// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldSingle matches snapshot 1`] = `
<Field
  actions={
    Object {
      "setItemProp": [Function],
      "validateOnChange": [Function],
    }
  }
  name="test-field"
  onChange={[Function]}
/>
`;
